scripts/install.sh

```
@@ -139,52 +140,134 @@ check_git() {
-            info "Installing pnpm..."
-            npm install -g pnpm
-            if ! command -v pnpm >/dev/null 2>&1; then
-                error "pnpm installation failed. Please install it manually:"
-                printf '  npm install -g pnpm\n'
-                exit 1
+check_node() {
+    NODE_BIN="$(find_node)"
+    if [ -n "$NODE_BIN" ]; then
+        node_version="$("$NODE_BIN" --version)"
+        success "Node.js $node_version ($NODE_BIN)"
+        export PATH="$(dirname "$NODE_BIN"):$PATH"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)